src/DynamoCoreWpf/ViewModels/Search/NodeSearchElementViewModel.cs

```diff
@@ -278,16 +279,22 @@ protected virtual void CreateAndConnectToPort(object parameter)
 
                 // Create a new node based on node creation name and connection ports
                 dynamoViewModel.ExecuteCommand(new DynamoModel.CreateAndConnectNodeCommand(id, initialNode.GUID,
-                    Model.CreationName, 0, Model.AutoCompletionNodeElementInfo.PortToConnect, adjustedX, 0, createAsDownStreamNode, false, true));
+                    Model.CreationName, 0, Model.AutoCompletionNodeElementInfo.PortToConnect, adjustedX, adjustedY, createAsDownStreamNode, false, true));
```

Comment:
I thought we made the adjusted Y 0 on purpose, @aparajit-pratap @zeusongit can you confirm? Maybe we thought the auto layout algorithm will simply solve that? Anyway, the solution here looks simple enough to be consistent

Reply:
There was some back-and-forth on this I remember. But I dont recall the problem. Anyway it seems to work fine.